assets/src/edit-story/components/canvas/pagemenu/pageMenuButton.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import {
+  Button,
+  BUTTON_VARIANTS,
+  BUTTON_TYPES,
+  BUTTON_SIZES,
+} from '../../../../design-system';
+import WithTooltip from '../../tooltip';
+
+function PageMenuButton({ children, title, shortcut, ...rest }) {
+  return (
+    <WithTooltip title={title} shortcut={shortcut}>
```

Comment:
Use `Tooltip` from the `design-system` instead?

Reply:
Yes, we should implement that too. But let's hold off on doing it piecemeal and implement it through somewhere. It's probably easiest to change the insides of `WithTooltip` instead actually. Noted for future ticket 📝